README.md

```diff
@@ -1,35 +1,30 @@
-Buildbot configuration for Halide build bots
+# Configuration for Halide buildbots
 
 To get started:
 
-Install python3.x, python-pip, then run:
+Install Python 3.6+, python3-pip, and python3-venv, then run:
```

Comment:
Well, `pathlib` won't work with Python 3.5, so I assume that it will fail at the import step. We could add a check (`sys.version_info` is a named tuple of the maj/min/patch versions).